### ISSUE TITLE
epl derivatives and bsd license

### ISSUE THREAD

**Issue by noahdesu** [usethesource/capsule#19]
I was curious about the statement in the README `Capsule was recently extracted from the usethesource/vallang project `. Is Capsule an EPL derivative work? IANAL, so but I was reading about EPL earlier and it seems much less permissive than BSD and I'm wondering about how users should reason about this.

**Comment by msteindorfer** [comment]
Hi @noahdesu, thanks for your inquiry. I agree that the statement from the README may be fuzzy, thus I'll clean it up as soon as possible. Anyways, Capsule is properly BSD licensed it not EPL derivative works. More specifically:

* In its beginning, most of Capsule was automatically generated by a code generator and temporary lived in `vallang` (I'm also one of the core developers of that project) before I decided to give the data structures their own repository (i.e., regenerating them into another repository). 
* Since I'm the sole developer (and thus sole copyright holder) of the Capsule sources, I decided to relicense the data structures under BSD.

At [usethesource.io](http://usethesource.io) (i.e., the umbrella under which Capsule is developed), we use BSD for new projects and keep using EPL for existing code bases. Since in the case of Capsule converting the license was feasible due to the split and the copyright situation, we did the conversion.
